src/tests/bdd/entity/validate_default_entity.feature

```diff
@@ -0,0 +1,117 @@
+
+    Given there exist document with id "1" in data source "data-source-name"
+    """
+    {
+      "name": "root_package",
+      "description": "",
+      "type": "dmss://system/SIMOS/Package",
+      "isRoot": true,
+      "content": [
+        {
+          "address": "$2",
+          "type": "dmss://system/SIMOS/Reference",
+          "referenceType": "storage"
+        }
+      ]
+    }
+    """
+    Given there exist document with id "2" in data source "data-source-name"
+    """
+    {
+      "name": "PersonBlueprint",
+      "type": "dmss://system/SIMOS/Blueprint",
+      "attributes": [
+        {
+          "name": "type",
+          "type": "dmss://system/SIMOS/BlueprintAttribute",
+          "attributeType": "string",
+          "optional": false
+        },
+        {
+          "name": "name",
+          "type": "dmss://system/SIMOS/BlueprintAttribute",
+          "attributeType": "string",
+          "label": "Name",
+          "optional": false
+        },
+        {
+          "name": "phoneNumber",
+          "type": "dmss://system/SIMOS/BlueprintAttribute",
+          "attributeType": "integer",
+          "label": "Phone number",
+          "optional": true,
+          "default": 123445566778
+        },
+        {
+          "name": "nationality",
+          "type": "dmss://system/SIMOS/BlueprintAttribute",
+          "attributeType": "string",
+          "label": "Nationality",
+          "optional": false,
+          "default": "Norwegian"
+        },
+        {
+          "name": "prime_minister",
+          "type": "dmss://system/SIMOS/BlueprintAttribute",
+          "attributeType": "dmss://data-source-name/root_package/PersonBlueprint",
+          "label": "Prime Minister",
+          "optional": true,
+          "default": {
+            "type": "dmss://data-source-name/root_package/PersonBlueprint",
+            "name": "Jonas",
+            "nationality": "Norwegian"
+          }
+        },
+        {
+          "name": "lngLat",
+          "type": "dmss://system/SIMOS/BlueprintAttribute",
+          "attributeType": "number",
+          "optional": true,
+          "dimensions": "*",
+          "contained": true,
+          "default": [
+            63.43, 10.39
+          ]
+        }
+      ]
+    }
+
+    """
+
+  Scenario: Validate existing simple example
+
+    Given i access the resource url "/api/documents/data-source-name/root_package/"
+    When i make a "POST" request with "1" files
+    """
+    {
+      "document": {
+        "_id": "Erna",
+        "type": "dmss://data-source-name/root_package/PersonBlueprint",
+        "name": "Erna",
+        "nationality": "Norwegian"
+      }
+    }
+    """
+    Then the response status should be "OK"
+
+    Given i access the resource url "/api/entity/validate-existing-entity/data-source-name/$Erna"
+    When i make a "POST" request
+    Then the response status should be "OK"
+    And the response should contain
+    """
+    "OK"
+    """
```

Comment:
I don't see why you need these tests. You want to test the default validation, and to do that you only need to test the blueprint

Reply:
No, agreed, it does not make sense to test this here. (removed)